sqlstore/src/main/resources/db/migration/V0030__project_shortName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD COLUMN shortName varchar(5) DEFAULT NULL;
```

Comment:
That could have non-alphanumeric characters in it. There's no `REGEX_REPLACE` for MySQL.